module/applications/actor/base-sheet.mjs

```diff
@@ -39,6 +39,15 @@ export default class ActorSheet5e extends ActorSheet {
 
   /* -------------------------------------------- */
 
+  /**
+   * Track the most recent drag event.
+   * @type {DragEvent}
+   * @protected
+   */
+  _event = null;
```

Comment:
I'm don't think storing the event like this is the best idea. I would suggest instead overriding the default `Actor#_onDropItem` method so that it passes the event through:

```js
  /** @override */
  async _onDropItem(event, data) {
    if ( !this.actor.isOwner ) return false;
    const item = await Item.implementation.fromDropData(data);
    const itemData = item.toObject();

    // Handle item sorting within the same Actor
    if ( this.actor.uuid === item.parent?.uuid ) return this._onSortItem(event, itemData);

    // Create the owned item
    return this._onDropItemCreate(event, itemData);
  }
```

I created an issue on core to hopefully get this change integrated there also: https://github.com/foundryvtt/foundryvtt/issues/8872

Reply:
Having it integrated in core would be ideal. I did override these in an earlier commit, but then realised it broke several functions on several actor types, notably tossing an error when dropping folders of documents. Is there a downside to leaving it as is as a temporary solution? I can add a 'TODO' comment to have `event` removed later on.

Reply:
I'll defer this to Kim's review.

Reply:
Agreed that having the store the event is not the ideal solution. I think foundryvtt/foundryvtt#8872 would be a breaking change (assuming we want to have the `event` be the first argument to match the rest of the API), so the earliest it can go in is v12. The implementation here with overriding `_onDrop` is only very minor tech debt though so it's probably ok as an interim solution with a TODO to change this later.